extensions/resteasy-classic/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -998,7 +998,12 @@ private static JaxrsProvidersToRegisterBuildItem getFilteredJaxrsProvidersToRegi
                 throw new RuntimeException("More than one Application class: " + applications);
             }
             selectedAppClass = applicationClassInfo;
-            // FIXME: yell if there's more than one
```

Comment:
Not sure if this was intended to be removed?

Reply:
It was intentional as this has already been addressed in the code above